tests/integrate_test/test_integrate_existent_scores.py

```diff
@@ -0,0 +1,379 @@
+# -*- coding: utf-8 -*-
```

Comment:
It is regarded to deploy SCORE also.
so I want to add test that about audit enable circumstances

Reply:
I added tests in audit enable circumstances.